lib/SpaInterface/SpaInterface.cpp

```diff
@@ -2,26 +2,20 @@
 
 #define BAUD_RATE 38400
 
-#if defined(ESP8266)
-    HardwareSerial spaSerial = Serial;
-#elif defined(ESP32)
-    HardwareSerial spaSerial = Serial2;
-#endif
-
 SpaInterface::SpaInterface(Stream &p) : port(p) {
```

Comment:
This code is redundant and can be removed.  The idea was to allow the serial stream to be passed in but we are now doing this via compile flags.